likecoin/js/admin-settings/src/App.js

```diff
@@ -1,31 +1,36 @@
-import './App.css';
-import { Route, Switch } from 'react-router-dom';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import { useSelect } from '@wordpress/data';
+import { SITE_LIKER_INFO_STORE_NAME } from './store/site-likerInfo-store';
+import Header from './components/Header';
+import MainSettingLayout from './pages/MainSettingLayout';
 import MainSettingPage from './pages/MainSettingPage';
-import LikecoinButtonPage from './pages/LikecoinButtonPage';
-import PublishSettingPage from './pages/PublishSettingPage';
-import OtherSettingPage from './pages/OtherSettingPage';
+import LikerIdSettingLayout from './pages/LikerIdSettingLayout';
+import AdvancedSettingPage from './pages/AdvancedSettingPage';
+import UserLikerIdSettingPage from './pages/UserLikerIdSettingPage';
+import SiteLikerIdSettingPage from './pages/SiteLikerIdSettingPage';
 import SponsorLikecoinPage from './pages/SponsorLikecoinPage';
+import LikeCoinHelpPage from './pages/LikeCoinHelpPage';
 
 function App() {
+  const {
+    DBUserCanEditOption,
+  } = useSelect((select) => select(SITE_LIKER_INFO_STORE_NAME).selectSiteLikerInfo());
   return (
-    <div className="App">
-      <Switch>
-        <Route path="/" exact>
-          <MainSettingPage />
+    <div className="wrap">
+      <h2> </h2>
```

Comment:
```suggestion
```

Reply:
This is `h2` under `.wrap` is needed for displaying notice correctly, will add comment